build.gradle

```diff
@@ -34,6 +34,20 @@ dependencies {
     compile 'org.json:json:20160810'
```

Comment:
Found a [helpful reference](https://docs.gradle.org/current/dsl/index.html) on the Domain-Specific Language that Gradle uses in its configuration scripts.